hydromt_wflow/wflow.py

```diff
@@ -417,6 +417,13 @@ def setup_rivers(
         """
         self.logger.info("Preparing river maps.")
 
+        # Check that river_upa threshold is bigger than the maximum uparea in the grid
+        if river_upa > self.grid[self._MAPS["uparea"]].max():
+            raise ValueError(
+                f"river_upa threshold {river_upa} should be larger than the maximum \
+uparea in the grid {self.grid[self._MAPS['uparea']].max()}"
```

Comment:
Found this one after testing locally, need to convert it to float to prevent this from happening:

```
ValueError: river_upa threshold 3000 should be larger than the maximum uparea in the grid <xarray.DataArray 'wflow_uparea' ()> Size: 8B
array(24.65899658)
Coordinates:
    spatial_ref  int32 4B 0
```